### PR TITLE
Modernize Admin Site

### PR DESCRIPTION
**Pull request by miketheman**
TL, DR: It's prettier!

Some high-level steps:
- Remove vendored code and replace with updated NPM packages
- Update themes from Bootstrap 3 to 4 (`box` vs `card`)
- Variety of tweaks to make things look anfd feel more natural - (e./g. use `-danger` for destructive actions)

Commits have been split to make reviewing sections simpler, since it can be a lot to consume.

Testing - I wasn't able to verify the malware or emails sections, since I don't have any records in my dev DB. If there's a way to create those on-demand (FactoryBoy classes yell at me outside of a test suite), pointers are welcome!

### CODE REVIEW

**Comment by miketheman** [comment]
I made a Loom to show some of the changes: [Admin · Warehouse — Private Browsing - 3 January 2023 - Watch Video](https://www.loom.com/share/e8803177bcf2438bb54d1f5572db53a8)

File: warehouse/admin/templates/admin/base.html [hunk 1]
```diff
@@ -29,170 +29,225 @@
...
   </div>
-
-
+  <!-- /.content-wrapper -->
+
+  <!-- Main Footer -->
+  <footer class="main-footer">
+    <!-- To the right -->
+    <div class="float-right d-none d-sm-inline">
+      Have you in fact got any cheese here at all?
```
Comment (ewdurbin) [inline warehouse/admin/templates/admin/base.html:244]: 😂

**Review by ewdurbin — APPROVED** [review]
this is really slick. thank you @miketheman.

**Comment by ewdurbin** [comment]
I'll leave some time for other @pypi/warehouse-admins to review before merge.

**Review by di — APPROVED** [review]
I didn't look at everything, but I didn't see anything that would change existing behavior. This is awesome!

**Pull request merged**